security.rst

```diff
@@ -2590,7 +2590,17 @@ Authentication Events
 
 .. raw:: html
 
-    <object data="./_images/security/security_events.svg" type="image/svg+xml"></object>
+    <object data="../_images/security/security_events.svg" type="image/svg+xml"
+        alt="A flow diagram showing the authentication events in a
+        request-response cycle. First, the authenticate method is called,
+        followed by the check passport event. The createToken method is
+        called after this event, followed by the authentication token
+        created and authentication success. Finally, the
+        onAuthenticationSuccess method is called, which results in a login
+        success event. Any authentication error between the authenticate
+        and on authentication success methods results in the login failure
+        event that is dispatched onAuthenticationFailure method is called."
```

Comment:
```suggestion
        event that is dispatched after the onAuthenticationFailure method is called."
```

Reply:
I'm not sure about adding ultra long descriptions to complex diagrams. It will make maintenance much harder (and it's already very hard) ... also, the text below explains all that with more detail. Thanks.